mathlibtools/lib.py

```diff
@@ -583,9 +586,8 @@ def build(self) -> None:
     def clean_mathlib(self, force: bool = False) -> None:
         """Restore git sanity in mathlib"""
```

Comment:
If you really want this function to do nothing while working on mathlib then it should be renamed to `clean_mathlib_dep` or something like this. I still don't understand your workflow, but I wouldn't mind merging this change.